### PR TITLE
Add Healthchecks

### PR DESCRIPTION
**Pull request by amjed-ali-k**
First of all, thank you for your contribution! 😄

### ☑️ Self Check before Merge

- [x] I have tested the template using the method described in README.md thoroughly
- [x] I have ensured that I put as much default values as possible (except passwords) to ensure minimum effort required for end users to get started.
- [x] I have ensured that I am not using the "latest" tag as this tag is dynamically changing and might break the one-click app. Use a fixed version.
- [x] I have made sure that instructions.start and instructions.end are clear and self-explanatory.
- [x] Icon is added as a png file to the logos directory.

### CODE REVIEW

File: public/v4/apps/healthchecks.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+            notExposeAsWebApp: 'true'
+caproverOneClickApp:
+    variables:
+        - id: $$cap_hc_version
+          label: hc Manifest digest
+          defaultValue: 'v1.25.0'
+          description: Find on https://hub.docker.com/r/healthchecks/healthchecks/tags
+        - id: $$cap__postgres_password
+          label: Postgres Password
+          description: Password must be at least 12 characters.  Please use a random string.
```
Comment (githubsaturn) [inline public/v4/apps/healthchecks.yml:41]: Let's use a `defaultValue: $$cap_gen_random_hex(16)` here for easier deployment

**Comment by githubsaturn** [comment]
Looks good! Just one minor comment.

File: public/v4/apps/healthchecks.yml (hunk 1 above)
Comment (amjed-ali-k) [reply public/v4/apps/healthchecks.yml:41]: done ✅

**Comment by githubsaturn** [comment]
Thanks!

**Pull request merged**